pkgs/applications/office/vnote/default.nix

```diff
@@ -0,0 +1,56 @@
+{ stdenv, fetchFromGitHub, qt5, hicolor-icon-theme, makeDesktopItem }:
+
+let
+  description = "A note-taking application that knows programmers and Markdown better";
+  desktopItem = makeDesktopItem {
+    name = "VNote";
+    exec = "vnote";
+    icon = "vnote";
+    comment = description;
+    desktopName = "VNote";
+    categories = "Office";
+  };
+in stdenv.mkDerivation rec {
+  version = "2.5";
+  name = "vnote-v${version}";
```

Comment:
Please use `pname = "vnote";` instead as `name = "${pname}-${version}";` is already set.

Reply:
fixed, will push shortly